core/src/main/java/com/klaytn/caver/wallet/keyring/KeyringFactory.java

```diff
@@ -250,6 +402,32 @@ public static RoleBasedKeyring createWithRoleBasedKey(String address, List<Strin
 
     /**
      * Decrypts a KeyStore json string and returns a keyring instance.
+     * <pre>Example :
+     * {@code
+     * String keyStoreStr = "{\n" +
+     *                 "  \"version\":3,\n" +
```

Comment:
I think you need to change this to version 4.

Reply:
I modified it.